apps/evm_family/evm_sign_msg.c

```diff
@@ -431,11 +458,13 @@ void evm_sign_msg(evm_query_t *query) {
   }
 
   if (NULL != sign_msg_ctx.msg_data) {
+    pb_release(EVM_SIGN_TYPED_DATA_STRUCT_FIELDS, &(sign_msg_ctx.typed_data));
```

Comment:
this is something our of ordinary. let's add a relevant background for the requirement of this explaining the purpose and need.

Reply:
updated